obc.py

```diff
@@ -23,7 +23,12 @@ def start(self):
 
         # start apps
         imageService.start()
+<<<<<<< HEAD
+        videoDisplay.start()
+        pixhawk.start()
+=======
         telemData.start()
```

Comment:
You're also gonna wanna take out this line as this is the "incoming changes", and we wanna keep your code which calls `pixhawk.start()`